lib/factory_girl/linter.rb

```diff
@@ -1,11 +1,12 @@
 module FactoryGirl
   class Linter
-    def self.lint!(factories_to_lint)
-      new(factories_to_lint).lint!
+    def self.lint!(factories_to_lint, options)
```

Comment:
Thanks. Given all that, I think just a separate class will make it cleaner, so it's just a separate feature. Revision on its way.